### DISCUSSION TITLE
Segmentation fault (core dumped)가 발생할 경우 transformers 패키지를 재설치해봅시다.

### DISCUSSION THREAD

**Discussion by ymnseol** [boostcampaitech4nlp2/level1_semantictextsimilarity_nlp-level1-nlp-13#50] [General]
한 줄 요약: AutoTokenizer를 이용한 decoding 중 Segmentation fault (core dumped) 발생 시 transformers 패키지 삭제 후 재설치

develop branch를 pull해서 train.py를 실행시킬 때 validation 시작 시 segmentation fault (core dumped)와 함께 실행이 중단되는 오류를 경험했는데요, 저의 경우 EDA > output_analysis.py > OutputEDA > appendf > self.tokenizer.decode(s)에서 문제가 발생했습니다😢

Segmentation fault가 발생할 때 별다른 오류 발생 위치를 알려주지 않습니다! 혹시나 비슷한 부분에서 오류를 겪고 있다면 패키지 uninstall 후 install해봅시다...

**Comment by intrandom5** [comment]
비슷하게 저는 'klue/roberta-large' is a correct model identifier listed on 'https://huggingface.co/models'...
이런 식으로 에러 뜨면서 모델이 불러와지지 않는 현상도 있었는데요.
>>>
    pip install transformers --upgrade
로 버전을 최신화 해주니까 해결됐습니다. uninstall하고 install하는 거랑 같은 원리일겁니다.
